src/compton2/Compton2.hh

```diff
@@ -0,0 +1,197 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   compton2/Compton2.hh
+ * \author Andrew Till
+ * \date   11 May 2020
+ * \brief  Header file for compton interface
+ * \note   Copyright (C) 2017-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//----------------------------------------------------------------------------//
+
+#ifndef rtt_compton2_Compton2_hh
+#define rtt_compton2_Compton2_hh
+
+// C++ standard library dependencies
+#include <string>
+#include <vector>
+// Draco libraries
+#include "compton2/Sparse_Compton_Matrix.hh"
```

Comment:
Remove this comment and let clang-format order these include directives.  The stdlib headers should be listed after draco headers.

Reply:
I'll check the squashed version to see what git-format did